installation.md

```diff
@@ -0,0 +1,176 @@
+python .\start_all_sensors.py
+```
+
+Everything should run without errors, otherwise you have to debug, check that the absolute paths are set correctly and
+so on.
+
+In a second terminal:
+
+```bash
+cd path\to\recordings\surgeon_recording
+conda activate surgeon_recording
+python .\recorder_app.py
+```
+
+Open a new browser tab and type `127.0.0.1:8080` in the URL field to access the web interface of the recorder app. Start
+a recording with the name `test_simulated`, stop it after a short time and check that the
+directory `path\to\recordings\surgeon_recording\data` contains a folder `test_simulated` that contains the non-empty
+files *camera.csv, depth.avi, emg.csv, optitrack.csv, rgb.avi, tps.csv*.
+
+### RealSense Camera
+
+9. Connect the camera to the computer. Make sure to use a fast USB cable and port because the camera data is
+   particularly heavy.
+
+10. Open the Intel RealSense Viewer and check if any firmware update are required and install them. Then, change to `2D`
+    view and turn on the stereo module to check that you are getting the RGB data. Close the viewer afterwards.
+
+11. Open the *Camera* app of Windows and check that you get the RGB image of the camera.
+
+### Motive
+
+12. Power up the portable OptiTrack camera system, connect it to the computer and launch Motive. Be aware that without
+    the OptiTrack plugged in, it is not possible to use Motive due to the lack of a license.
+
+13. Find the *Streaming Pane* in the toolbar (the icon looks like a signal / antenna) and activate the first option
+    called *Broadcast Frame Data*.
+
+14. Define the desired rigid bodies by selecting markers and creating a rigid body from them. Then, go to the *Assets
+    Pane* and edit the rigid bodies. Give them a name, a streaming ID, and put something around 30 for the smoothing
+    value.
+
+15. In the [sensor parameters file](source/surgeon_recording/config/sensor_parameters.json), put the rigid bodies into
+    the list `frames` under `optitrack` with the corresponding labels and streaming IDs.
+
+### Second Test
+
+After step 16, let's do another test. Follow the instructions from [the first test](#first-test) (with a different
+folder name, e.g. `test_cameras`), except that in the sensor parameters file you put `on` in the status of the camera
+and optitrack. In the data folder, the *optitrack.csv, rgb.avi, depth,avi* files should now contain real data.
+
+**IMPORTANT**: Motive has to be running in the background for this to work.
+
+### TPS
+
+16. With the explorer, go to `path\to\unzipped\folders\Chameleon Installer-EPFL1 & 2` and
+    run `Install_Chameleon_TVR.exe`. Continue with the installation even if Windows complains and leave the default
+    values as is.
+
+17. Plug in the red USB receiver with number 184A57, turn on the transmitter box with number two, and connect a finger.
+    Run Chameleon as admin and choose configuration *EPFL-S2*, check that the connection is established successfully and
```

Comment:
you should add instructions on how to always run it as asdmin. I did it on my computer and that saves a lot of hassle